SCClassLibrary/Platform/windows/WindowsPlatform.sc

```diff
@@ -43,8 +43,9 @@ WindowsPlatform : Platform {
 
 	defaultTempDir {
 		// +/+ "" looks funny but ensures trailing slash
```

Comment:
This is now outdated. We can either remove it, or replace with "// ensure trailing slash due to backwards compatibility" similarly to macOS implementation?

Reply:
done, thanks!